### ISSUE TITLE
[mobile ux] 4. Order cycle design update

### ISSUE THREAD

**Issue by daniellemoorhead** [openfoodfoundation/openfoodnetwork#4572]
## What is the problem we are solving 
Currently users do not notice the order cycle section as they are not used to ordering in cycles. 

## Success factors = expected outcome 
- [ ] Users start to notice that the order cycle section and therefore there is a decrease in displaying the 'select an order cycle' message

## Design overview
**HOW IT LOOKS NOW**
_Mobile_
<img width="477" alt="Screen Shot 2019-12-13 at 4 24 03 pm" src="https://user-images.githubusercontent.com/10949052/70771385-03ef5200-1dc5-11ea-880f-98c69018b48b.png">

_Tablet_
<img width="1183" alt="Screen Shot 2019-12-13 at 4 23 38 pm" src="https://user-images.githubusercontent.com/10949052/70771402-136e9b00-1dc5-11ea-8d60-ededfcae9e91.png">

_Desktop_
<img width="706" alt="Screen Shot 2019-12-13 at 4 23 50 pm" src="https://user-images.githubusercontent.com/10949052/70771392-09e53300-1dc5-11ea-849d-40ca38fb9833.png">

**HOW IT WILL LOOK IN FUTURE**
_Mobile_
<img width="244" alt="Screen Shot 2019-12-13 at 4 25 48 pm" src="https://user-images.githubusercontent.com/10949052/70771482-5af52700-1dc5-11ea-9226-715c4e45cf73.png">

_Tablet_
<img width="472" alt="Screen Shot 2019-12-13 at 4 26 05 pm" src="https://user-images.githubusercontent.com/10949052/70771487-5e88ae00-1dc5-11ea-952e-582766aca738.png">

_Desktop_
<img width="779" alt="Screen Shot 2019-12-13 at 4 26 19 pm" src="https://user-images.githubusercontent.com/10949052/70771488-621c3500-1dc5-11ea-854f-aabc47f93740.png">

_Design for different states_
![OC Selector](https://user-images.githubusercontent.com/19785399/76047421-9a4c9300-5f30-11ea-893e-09a1e01064dc.png)

Design details are kept in our Zeplin account: https://app.zeplin.io/project/59532764a6c78ff3c534b372/dashboard

**Comment by daniellemoorhead** [comment]
@RachL is there any way we can see that this design fixes the issue shoppers have in understanding that they have to select an order cycle?

**Comment by RachL** [comment]
@daniellemoorhead 

I think so yes.

Automatically:
we cannot add events yet with our Matomo set up, but we have heatmaps.  I just need to get our limit lifted up because currently I consumed everything by testing it 🙈 

![image](https://user-images.githubusercontent.com/6525576/70786031-f8336980-1d8b-11ea-853e-30f4540f10cd.png)

With our users:
Each week I have complaints on this. So I have a 5-6 shops I can ask to follow the difference after the new design is in place. I'm sure other instance have it too. 
It's almost the only thing I hear complaints about in terms of design...

Also I'm not sure the version with the carret and a blue background is correct.... I think each time you have several OC open you have the red box.

**Comment by daniellemoorhead** [comment]
> I think each time you have several OC open you have the red box.

@yukoosawa @mkllnk we'll need to understand the different states that this order cycle section has, so that we cover them all. Seems there's one additional one that @RachL has identified that we haven't included in the list. @mkllnk could you please confirm the full list so that @yukoosawa can make sure she covers all of them? :)

**Comment by daniellemoorhead** [comment]
When doing this @mkllnk let's look at all the states that the OC can be in and see if a change needs to happen around the use case that @RachL raises.

**Comment by luisramos0** [comment]
I am planning to tackle the remaining 3 issues in this epic now.

**Comment by daniellemoorhead** [comment]
Time for a design sweep on this epic, to make sure @yukoosawa is happy with the output, that we've achieved the desired outcomes, and that we can officially close this!

**Comment by daniellemoorhead** [comment]
## DESIGN SWEEP 

**No open OC**
- [ ] closed message needs to span the window

**Comment by yukoosawa** [comment]
## DESIGN SWEEP - Sunday May 10, 2020

**_Google doc w/images to accompany issues: https://docs.google.com/document/d/1JsplLJJLhRwLUKv7WrgEr6thOylOtJ3JMC5wni42gEE/edit?usp=sharing_**

**OC Selector**
- [ ] Increase width of OC selector dropdown so more characters can fit in [desktop & tablet]
- [ ] Widen OC selector dropdown on RIGHT SIDE so it has equal margins on left and right sides [mobile]
- [ ] Add padding to the left of the dropdown caret/arrow so it’s not positioned directly next to the text [all breakpoints]
- [ ] OC close text should be vertically centred with dropdown [tablet]
- [ ] Add more padding above the dropdown so it is vertically centered in the coloured bar [tablet]

**No open OC**
- [ ] Grey background behind closed message should span the viewport (I believe Matt has solved this with #4604) [desktop]
- [ ] Exclamation point ‘!’ and ‘Orders are closed’ text is not vertically centred with the shape [mobile]

**Comment by luisramos0** [comment]
great @yukoosawa thanks a lot.
I'll take this one. I'll make a new PR to fix these things :+1:

**Comment by luisramos0** [comment]
I have ticked the 4 items addressed in #5407

According to browserstack the two last "OC Selector" items are specific to ipad mini 3, it looks good on ipad and also on ipad mini 2019. I cant fix it because the simulator of ipad mini 3 is not working with my local.

**Comment by daniellemoorhead** [comment]
@Matt-Yorkley the first item in @yukoosawa's [OC design sweep document](https://docs.google.com/document/d/1JsplLJJLhRwLUKv7WrgEr6thOylOtJ3JMC5wni42gEE/edit?usp=sharing) needs your attention, not sure if you'd be alerted about this so pinging you in here: 

![Screen Shot 2020-05-15 at 11 49 36 am](https://user-images.githubusercontent.com/10949052/82002987-9791bb00-96a2-11ea-87eb-4aefc908f895.png)

**Comment by luisramos0** [comment]
Unassigning myself from this one.

There are 3 pending points after #5407 I believe:
- ipad mini 3 padding issue
- improve what's in #5407 about widening the OC selector (see @daniellemoorhead previous post)
- new item in Yuk's comment above: break text in two lines when OC description is long

**Comment by daniellemoorhead** [comment]
Ok, looking at this with Yuko and there are 2 things remaining to be done that can be viewed on the [design sweep doc](https://docs.google.com/document/d/1JsplLJJLhRwLUKv7WrgEr6thOylOtJ3JMC5wni42gEE/edit?usp=sharing) (look for the RED coloured sections)

- [ ] Vertical alignment issues on iPads

- [ ] Text wrapping in the field when there is a single OC and the text spans more than the field width.

These two things are all that remain to be fixed.

**Comment by luisramos0** [comment]
Awesome, thanks for clarifying @daniellemoorhead :ok_hand: 
The first one is page 4 in the doc. And the second one is the last one on the doc (page 8/9).
The doc: https://docs.google.com/document/d/1JsplLJJLhRwLUKv7WrgEr6thOylOtJ3JMC5wni42gEE/edit?usp=sharing

**Comment by luisramos0** [comment]
I am copying the content to the issue so new contributors dont need to navigate the doc:
Vertical alignment issue:
![image](https://user-images.githubusercontent.com/1640378/95191286-74d67f00-07c8-11eb-8887-10713e3c01fc.png)
![image](https://user-images.githubusercontent.com/1640378/95191302-7b64f680-07c8-11eb-9139-0e486e917491.png)
Text wrapping issue:
![image](https://user-images.githubusercontent.com/1640378/95191353-8e77c680-07c8-11eb-9d81-d4e978adf0d8.png)
![image](https://user-images.githubusercontent.com/1640378/95191364-920b4d80-07c8-11eb-81e6-72804827a0fb.png)

**Comment by luisramos0** [comment]
I tagged good first issue. This is a good first issue if you are an experienced CSSer.

**Comment by RachL** [comment]
Oh my oh my THE LAST ISSUE WAS MERGED

![ ](https://media.giphy.com/media/YFFKkihxcDRvI5ZhNp/giphy.gif)
